### DISCUSSION TITLE
How to rename existing files?

### DISCUSSION THREAD

**Discussion by homeisfar** [gpodder/gpodder#1547] [General]
According to https://github.com/gpodder/gpodder/issues/678 and the source with the rename_download extension I should be able to right click something somewhere to rename existing downloads. But I cannot find this option anywhere.

I added the extension to ~/gPodder/Extensions and enabled it in the preferences, but I cannot find a way to interact with this functionality from gPodder itself.

**Comment by auouymous** [comment]
There should be an item in the `Extras` menu.

The feature is included with the last two versions of gpodder, and doesn't need to be manually installed. If you are running it on an older version, try looking in the logs to see if the extension is crashing. Or maybe gpodder is selecting the extension installed by the system over the one you installed, try renaming yours and then enable it. Be sure to disable the other one in gpodder.

**Reply by homeisfar** [reply]
Oh, I see it now.

For the future if someone is wondering, I think the correct steps are:

1. Program preferences -> Extensions
2. Check the Rename Episodes after Download box, thereby enabling this extension.
3. (OPTIONAL) Press Edit Config and check a few options about the rename extension to your liking
4. Click the Extras drop menu on the main program and select rename all downloaded episodes.
